.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
 - 'node'
+- '8'
```

Comment:
Until Node.js v9 is released (October) Node.js v8 is covered by the `node` entry already.